addons/website/tests/test_ui.py

```diff
@@ -202,3 +202,9 @@ def test_08_website_style_custo(self):
 
     def test_09_website_edit_link_popover(self):
         self.start_tour("/", "edit_link_popover", login="admin")
+
+    def test_10_website_add_menu_link(self):
+        self.start_tour("/", "add_menu_link", login="admin")
+
+    def test_11_edit_megamenu(self):
```

Comment:
I was thinking since they test a different feature each. I could combine them but I thought it could be easier if we came to change the way a MegaMenu is added to the website, it would be easy to change that test.

However maybe having them in one single file could make it easier than having to navigate a dozen. I'll follow your call!

> On the other hand, editing a menu link from the "edit link" button does not seem to work, but I guess it can be taken care of in another task
I think I found the culprit and it should be fix as well now.